recipe/meta.yaml

```diff
@@ -57,45 +26,57 @@ build:
 requirements:
   build:
     - python  # [win]
-    - freetype 2.6.*  # [unix]
-    - fontconfig 2.11.*  # [linux]
-    - freeglut  # [linux]
+    - perl
+    - freetype 2.7|2.7.*
+    - fontconfig 2.12.*
+    #- pango  # [linux]
+    #- freeglut  # [linux]
```

Comment:
Qt4 and Qt5 have different sets of dependencies, so several of the old ones don't apply for Qt5.